lib/freedom_patches/sidekiq.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Sidekiq
```

Comment:
Could we add a note about deleting this freedom-patch once we remove the MigrateSidekiqJobs migration in ~6 months?